ApiBundle/Controller/ContentController.php

```diff
@@ -113,7 +114,7 @@ public function listAction(Request $request)
         );
```

Comment:
for filter the contents according they are linked to a site, should update the mapping with 
'site_linked' => array('key' => 'siteLinked', 'type' => 'boolean'